pallets/dkg-proposals/src/lib.rs

```diff
@@ -227,43 +226,20 @@ pub mod pallet {
 	pub type ProposerThreshold<T: Config> =
 		StorageValue<_, u32, ValueQuery, DefaultForProposerThreshold>;
 
-	/// Proposer Set Update Proposal Nonce
-	#[pallet::storage]
-	#[pallet::getter(fn proposer_set_update_proposal_nonce)]
-	pub type ProposerSetUpdateProposalNonce<T: Config> = StorageValue<_, u32, ValueQuery>;
-
 	/// Tracks current proposer set
 	#[pallet::storage]
 	#[pallet::getter(fn proposers)]
-	pub type Proposers<T: Config> = StorageMap<_, Blake2_128Concat, T::AccountId, bool, ValueQuery>;
+	pub type Proposers<T: Config> =
+		StorageValue<_, BoundedVec<T::AccountId, T::MaxProposers>, ValueQuery>;
 
-	/// Tracks current proposer set external accounts
-	/// Currently meant to store Ethereum compatible 64-bytes ECDSA public keys
+	/// Tracks current ECDSA voting keys for each validator
+	pub type VotingKey<T> = BoundedVec<u8, <T as Config>::VotingKeySize>;
+	pub type VotingKeyTuple<T> = (<T as frame_system::Config>::AccountId, VotingKey<T>);
+	pub type VoterList<T> = BoundedVec<VotingKeyTuple<T>, <T as Config>::MaxProposers>;
```

Comment:
Are these supposed to be storage?

Reply:
No they're for clippy to simplify the complex type.